doc/secrets.md

```diff
@@ -0,0 +1,48 @@
+# BitSwan Secrets Management
+
+## Overview
+
+BitSwan uses environment files to manage secrets and sensitive configuration values. These secrets are made available as environment variables within pipelines and Jupyter notebooks.
+
+## Setup
+
+1. You will see a `secrets` directory in your VSCode Server workspace.
+2. Create environment files in the secrets directory.
+3. Configure secret groups in `pipelines.conf`.
+
+## Usage Example
+
+### 1. Create Environment File
+
+Create a file in the secrets directory (e.g., `foo`):
+```txt
+FOO=foo
+```
+
+### 2. Configure Secret Groups
+
+In `pipelines.conf`, add the secrets group:
+
+```ini
+[secrets]
+groups=foo
```

Comment:
if I understand right, each group gets an env file in secrets directory of the same name